Firestore/core/src/firebase/firestore/model/mutation.h

```diff
@@ -259,6 +280,12 @@ class Mutation {
         const absl::optional<MaybeDocument>& base_doc,
         const Timestamp& local_write_time) const = 0;
 
+    virtual absl::optional<ObjectValue> ExtractBaseValue(
+        const absl::optional<MaybeDocument>& maybe_doc) const {
+      (void)maybe_doc;
```

Comment:
`ABSL_ATTRIBUTE_UNUSED` has [no definition for MSVC](https://github.com/firebase/firebase-ios-sdk/blob/master/Firestore/third_party/abseil-cpp/absl/base/attributes.h#L545) so it's not portable enough.

There are other ways to do this but https://stackoverflow.com/a/1486931 suggests that we're doing the state of the art here.

But actually, since this is unconditionally unused, removing it from the signature is probably best. I've done that instead.